pingdom/resource_pingdom_maintenance_window.go

```diff
@@ -0,0 +1,242 @@
+package pingdom
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
```

Comment:
Can you change this to use github.com/hashicorp/terraform-plugin-sdk/helper/schema?